mac/deploy_mac.sh

```diff
@@ -73,7 +73,7 @@ build_installer_image()
     brew extract --version="${create_dmg_version}" create-dmg homebrew/cask
     brew install /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Formula/create-dmg@"${create_dmg_version}".rb
     # Get Jamulus version
-    local app_version="$(grep -oP 'VERSION = \K\w[^\s\\]*' Jamulus.pro)"
+    local app_version="$(sed -nE 's/^VERSION *= *(.*)$/\1/p' Jamulus.pro)"
```

Comment:
I think to be consistent with earlier versions and the rest of the script, this should use `${project_path}`. Line 98 below doesn't actually check the directory we are running in, as `${project_path}` is an absolute path, so the `-f` test always succeeds.
```suggestion
    local app_version="$(sed -nE 's/^VERSION *= *(.*)$/\1/p' ${project_path})"
```

Reply:
> I think to be consistent with earlier versions and the rest of the script, this should use ${project_path}

Thanks, changed. We should wait for another CI run though.

> Line 98 below doesn't actually check the directory we are running in, as ${project_path} is an absolute path, so the -f test always succeeds.

Yeah, but I'd rather not touch this as part of this PR. I plan to submit my other autobuild changes really soon.

Reply:
> > I think to be consistent with earlier versions and the rest of the script, this should use ${project_path}
> 
> Thanks, changed. We should wait for another CI run though.

Yes, I'd like to test install the Mac artifact with the new `create-dmg`. Unfortunately, non-release runs don't make the artifacts available until the last job has finished, afaik.

> > Line 98 below doesn't actually check the directory we are running in, as ${project_path} is an absolute path, so the -f test always succeeds.
> 
> Yeah, but I'd rather not touch this as part of this PR. I plan to submit my other autobuild changes really soon.

Sure. My point was only to illustrate why it was important to use `${project_path}` instead of a simple filename, as it doesn't actually ensure we are in the correct directory.